packages/kbn-test/jest_integration/jest-preset.js

```diff
@@ -15,9 +15,8 @@ module.exports = {
     (pattern) => !pattern.includes('integration_tests')
   ),
   setupFilesAfterEnv: [
+    ...preset.setupFilesAfterEnv,
```

Comment:
Do we need all these files as part of the base integration config?

Reply:
I think it's reasonable to expect that the jest preset for integration tests and unit tests of the "browser" variety would function mostly identical.